test/Unit/Cache/CacheStrategyConfiguratorTest.php

```diff
@@ -36,7 +36,10 @@ protected function setUp(): void
 
     public function testConfigureCache(): void
     {
-        $this->apiCacheMap->getCacheableApis('/api/')->shouldBeCalled();
+        $apiCacheMap = new ApiCacheMap;
+        $this->apiCacheMap
+            ->getCacheableApis('/api/')->shouldBeCalled()
+            ->willReturn($apiCacheMap->getCacheableApis('/api/'));
```

Comment:
Curious what the reason for this change was.

Reply:
I noticed that few lines weren't being covered, so this adds some valid data in the loop

All creating new instances, if we are to do more robust assertions we'll have to add some factories which I don't think is worth it